rox_description/urdf/xacros/diff_drive.xacro

```diff
@@ -8,28 +8,28 @@
     <xacro:diff_caster name="caster_wheel_front_right" parent="${parent}"
         vis_rot="-${pi/2} 0 ${pi/2}" vis_pos="0 0 0" collision_pos="0.045 0 0"
         joint_type="fixed">
-        <origin rpy="0 0 0" xyz="${0.2045 + frame_len + 0.045} -0.220 0.124"/>
+        <origin rpy="0 0 0" xyz="${0.2045 + 0.045} -0.220 0.124"/>
```

Comment:
now we don't have the frame length, can you give the summed up value? `${0.2045 + 0.045}`